docs/devel/integration.rst

```diff
@@ -40,16 +40,17 @@ translations to match the latest version of what it is you want to translate.
 Fetch changes made in Weblate
 +++++++++++++++++++++++++++++
 
-Weblate represents a branch from your VCS upstream that you can fetch changed
-strings from just like any other.
+Weblate represents a branch from your version control system upstream that
+you can fetch changed strings from just like any other.
 
 Either fetch the filesystem, or use :ref:`git-exporter`. Commit any pending
 changes first (see :ref:`lazy-commit`) from the user interface
 (in the :guilabel:`Repository maintenance`) or from the command-line
 using :ref:`wlc`.
 
-Automation is possible by granting the Weblate user push access to your
-repository and configuring :ref:`component-push` in the :ref:`component`.
+Automation is possible by granting the Weblate user (available for various
+version control systems) push access to your repository and configuring
```

Comment:
```suggestion
Automation is possible by granting the Weblate user (available on various
code hosting sites) push access to your repository and configuring
```

Reply:
I even read this the wrong way around to comply with what I thought I wrote ;)